src/coreclr/gc/handletablepriv.h

```diff
@@ -942,3 +942,7 @@ void CALLBACK BlockVerifyAgeMapForBlocks(PTR_TableSegment pSegment, uint32_t uBl
 PTR_TableSegment CALLBACK xxxAsyncSegmentIterator(PTR_HandleTable pTable, TableSegment *pPrevSegment, CrstHolderWithState *pCrstHolder);
 
 /*--------------------------------------------------------------------------*/
+
+#ifndef DACCESS_COMPILE
+int WhichGeneration(_UNCHECKED_OBJECTREF obj);
```

Comment:
A comment here is likely warranted. The policy decision being made here is something that could be confusing, especially since this is in the handle table header.